pkg/scheduler/plugins/nodenumaresource/pod_eventhandler.go

```diff
@@ -30,9 +33,11 @@ type podEventHandler struct {
 }
 
 func registerPodEventHandler(handle framework.Handle, cpuManager CPUManager) {
-	handle.SharedInformerFactory().Core().V1().Pods().Informer().AddEventHandler(&podEventHandler{
+	podInformer := handle.SharedInformerFactory().Core().V1().Pods().Informer()
+	eventHandler := &podEventHandler{
 		cpuManager: cpuManager,
-	})
+	}
+	frameworkexthelper.ForceSyncFromInformer(context.TODO().Done(), handle.SharedInformerFactory(), podInformer, eventHandler)
```

Comment:
While the function is being called, there may be other pod Update and Delete events，how to guarantee their order?

Reply:
Event Order will not change. The reason is that forceSyncEventHandler.waitForSyncDone blocks the entire EventHandler. When waitForSyncDone returns, the original accumulated Events will be consumed.